ansible/roles/magnum/templates/magnum.conf.j2

```diff
@@ -66,7 +66,11 @@ ca_file = {{ openstack_cacert }}
 
 [nova_client]
 region_name = {{ openstack_region_name }}
+{% if magnum_capi_helm_driver_enabled %}
+endpoint_type = publicURL
+{% else %}
 endpoint_type = internalURL
+{% endif %}
```

Comment:
This can be removed if https://github.com/stackhpc/magnum-capi-helm/pull/51 is accepted